packages/containerd/containerd-cri-base-json

```diff
@@ -0,0 +1,151 @@
+{
+    "ociVersion": "1.0.2-dev",
+    "process": {
+        "user": {
+            "uid": 0,
+            "gid": 0
+        },
+        "cwd": "/",
+{{#if settings.oci-defaults.capabilities}}
+        "capabilities": {
+            {{oci_defaults "capabilities" settings.oci-defaults}}
+        },
+{{/if}}
+{{#if settings.oci-defaults.resource-limits}}
+        "rlimits": [
+{{oci_defaults "resource-limits" settings.oci-defaults}}
```

Comment:
look into: is there a helper that wraps values in quotes or combines with commas?